pkg/filesystem/block_device_backed_file_pool_test.go

```diff
@@ -309,4 +309,12 @@ func TestBlockDeviceBackedFilePool(t *testing.T) {
 		sectorAllocator.EXPECT().FreeList([]uint32{0, 0, 0, 0, 0, 0, 0, 0, 5})
 		require.NoError(t, f.Close())
 	})
+
+	t.Run("WriteAt", func(t *testing.T) {
+		f, err := pool.NewFile()
+		require.NoError(t, err)
+
+		_, err = f.WriteAt([]byte{0}, -1)
+		require.Error(t, err, "Expected an OOB error for negative offset")
```

Comment:
Could you do a testutil.RequireEqualStatus() here instead? Just so we can make sure we got the string formatting of the error message right. Thanks!

Reply:
Sure- I wasn't sure if you wanted to check for a specific error, but checking for the status is a good compromise.